functions/exercise26/scriptSpec.js

```diff
@@ -0,0 +1,11 @@
+/// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals getLongestSubstringWithoutRepeatingCharacters*/
+
+describe(" Find longest substring in a given a string without repeating characters", (() => {
+    
+         it("If the string consists of one word 'elephant', then the longest substring without repeating characters is 'lphant'", (() => {
+             let substringWithoutRepeatingCharacters = getLongestSubstringWithoutRepeatingCharacters("elephant");
+             expect(substringWithoutRepeatingCharacters).toEqual("lphant");
```

Comment:
It is not correct, the longest is phant. There is no such substring lphant as l occurs before e that repeats

Reply:
L do not repeat. Example from the exercise: "google.com"; 
"example.com";                                       Result:"gle.com","xample.co"

Reply:
That is, in this exercise, the point is that the letters that are repeated are only once

Reply:
the result should be an lephant